apis/Google.Cloud.VideoIntelligence.V1/Google.Cloud.VideoIntelligence.V1/VideoAnnotationResultsException.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using System;
+
+namespace Google.Cloud.VideoIntelligence.V1
+{
+    /// <summary>
+    /// An error occurring when annotating an video.
+    /// </summary>
+    public sealed class VideoAnnotationResultsException : Exception
```

Comment:
For consistency with Vision, I'll rename this to AnnotateVideoException in a follow-up PR.
(The Vision API has the wrong filename for the class too, which I'll fix.)